test/test_other.py

```diff
@@ -8758,7 +8758,7 @@ def test_codesize_minimal(self, *args):
 
   @node_pthreads
   def test_codesize_minimal_pthreads(self):
```

Comment:
Maybe rename this to add `_strict`?

Reply:
The `test_codesize_minimal` test above has `STRICT` too without including it in the name, I think its fair to assume that most minimal codesize tests include these flags.